cogs/customs.py

```diff
@@ -40,70 +30,77 @@ def __init__(self, bot):
                               url='https://www.youtube.com/watch?v=I9GcnRAdY2M')
```

Comment:
add a TODO here to move this to be DB driven

Reply:
this actually looks like it was made for a specific and horrific event in our guild, i dont think it really has a place here anymore ?

I am planning to make the ability for custom commands too